Rewatch.lua

```diff
@@ -778,7 +799,7 @@ function rewatch_UpdateBar(spellName, player, stacks)
 	if(rewatch_bars[playerId][spellName]) then
 	
 		-- get buff duration
-		local a = select(7, UnitBuff(player, spellName, nil, "PLAYER"));
+		local a = select(6, AuraUtil.FindAuraByName(spellName, player, "PLAYER"));
```

Comment:
Oeh, what is this AuraUtil? The patch I have for this change is looping over `UnitBuff(player, counter, filter)` where counter is incremented from 1 to 40, your solution seems more efficient? Can you clarify?

Reply:
It's a new Bfa function found in FrameXML/AuraUtil.lua. It 'Finds the first aura that matches the name'. 
There are some important notes to bear in mind:
* '[A]ura names are not unique!'
* '[A]ura names are localized, what works in one locale might not work in another consider that in English two auras might have different names, but once localized they have the same name, so even using the localized aura name in a search it could result in different behaviour'.
* '[T]he unit could have multiple auras with the same name, this will only find the first'.

As this is used only for class spells, if I'm not mistaken, I don't believe we are likely to run into any problems.

See [WowPedia](https://wow.gamepedia.com/API_AuraUtil.FindAuraByName).